MaybeGame/EnemyConfig.cs

```diff
@@ -0,0 +1,49 @@
+﻿namespace MaybeGame
+{
+    public static class EnemyConfig
+    {
+        public static Enemy CreateZombie()
+        {
+            var enemy = new Enemy();
```

Comment:
If setting stats is all you do during object initialization, pass it to constructor instead